packages/portfolio/src/portfolio-routes.tsx

```diff
@@ -1,20 +1,25 @@
+import { useDbAccountActive } from '@workspace/db-react/use-db-account-active'
+import { useDbNetworkActive } from '@workspace/db-react/use-db-network-active'
 import { UiPage } from '@workspace/ui/components/ui-page'
-import { PortfolioRoutesLoaded } from './portfolio-routes-loaded.tsx'
-import { PortfolioUiAccountGuard } from './ui/portfolio-ui-account-guard.tsx'
-import { PortfolioUiNetworkGuard } from './ui/portfolio-ui-network-guard.tsx'
+import { useRoutes } from 'react-router'
+import { PortfolioFeatureIndex } from './portfolio-feature-index.tsx'
+import { PortfolioFeatureTx } from './portfolio-feature-tx.tsx'
+
+export function Router() {
+  const account = useDbAccountActive()
+  const network = useDbNetworkActive()
+
+  return useRoutes([
+    { element: <PortfolioFeatureIndex account={account} network={network} />, path: '*' },
+    { element: <PortfolioFeatureTx network={network} />, path: 'tx/:signature' },
+    // { element: <div>Page not found :(</div>, path: '*' },
+  ])
+}
 
 export default function PortfolioRoutes() {
   return (
     <UiPage containerProps={{ className: 'sm:max-w-2xl' }}>
-      <PortfolioUiNetworkGuard
-        render={({ network }) => (
-          <PortfolioUiAccountGuard
-            render={({ account }) => {
-              return <PortfolioRoutesLoaded account={account} network={network} />
-            }}
-          />
-        )}
-      />
+      <Router />
```

Comment:
We no longer need this wrapped `Router` component. it can all live in the `PortfolioRoutes` component.

Reply:
Updated